ENTITY/GetEntityType.md

```diff
@@ -8,15 +8,22 @@ ns: ENTITY
 int GET_ENTITY_TYPE(Entity entity);
 ```
 
-```
-Returns:
+Gets the type of entity for a given entity.
+
+## Parameters
+* **entity**: 
+
+## Return value
+the entity type of the given entity as a integer.
+
 0 = no entity
 1 = ped
 2 = vehicle
 3 = object
-```
 
-## Parameters
-* **entity**: 
+## Examples
 
-## Return value
+```lua
+local light = GetEntityModel(prop_streetlight_01)
```

Comment:
You should explain how to get the entity handle here before you pass it, and the `lightType` variable name makes me think there's multiple types of lights😅

Also not sure why you're getting the model, `GetEntityType` takes an entity handle.

So I would write this the following way:

```lua
local entityType = GetEntityType(PlayerPedId()) -- entityType should be '4'
```